Mathlib/AlgebraicGeometry/Modules/Tilde.lean

```diff
@@ -188,16 +200,16 @@ theorem res_apply (U V : Opens (PrimeSpectrum.Top R)) (i : V ⟶ U)
 
 lemma smul_section_apply (r : R) (U : Opens (PrimeSpectrum.Top R))
     (s : (tildeInModuleCat M).obj (op U)) (x : U) :
-    (r • s).1 x = r • (s.1 x) := rfl
+    (r • s).1 x = r • (s.1 x) :=
```

Comment:
Can this be salvaged later?

Reply:
I'm not sure. Probably yes. But this isn't a really useful defeq to have.